### ISSUE TITLE
Be able to accept an empty array as input

### ISSUE THREAD

**Issue by jeffharrington** [kellyjonbrazil/jtbl#16]
## Description
I notice that `jtbl` throws an error when it receives an empty list as input (`[]`). 

This feels like a bug since en empty list is considered valid json. 

## Version 
```
jtbl:   version 1.5.1
```

## Steps to recreate
```shell
echo "[]" | jtbl
```

## Expected Output
An empty response (similar to what is returned when you send an empty object `{}`. 

## Observed Output
```
Traceback (most recent call last):
  File "/opt/homebrew/bin/jtbl", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/jtbl/cli.py", line 339, in main
    succeeded, json_data = check_data(json_data, columns=columns)
    ^^^^^^^^^^^^^^^^^^^^
```

**Comment by kellyjonbrazil** [comment]
Thanks for reporting this. Yes, that is ugly! I'll get this fixed and add a test in the next release.

**Comment by kellyjonbrazil** [comment]
fixed in v1.5.2
